do-not-commit/exit.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+if False:
+    # ok
+    sys.exit(2)
+
+if True:
+    # ruleid: use-sys-exit
+    exit(3)
+
+
+def check_db(user):
+    if user is None:
+        # ruleid: use-sys-exit
+        exit(4)
```

Comment:
Use `sys.exit` over the python shell `exit` built-in. `exit` is a helper for the interactive shell and may not be available on all Python implementations. https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python ([.exit.use-sys-exit](https://semgrep.dev/r/.exit.use-sys-exit))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=.exit.use-sys-exit) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>